blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { createRef, useCallback, useEffect, useReducer, useState } from "react";
-import { Button, BUTTON_STYLES, BUTTON_TYPES } from "@wpmedia/shared-styles";
+import { Button, Stack } from "@wpmedia/arc-themes-components";
 import { useContent } from "fusion:content";
 
 import ResultItem from "./result-item";
 import { reduceResultList } from "./helpers";
 
+const BLOCK_CLASS_NAME = "b-result-list";
```

Comment:
Plural change - also I recommend we use `b-results-list__item` for styling each item in the results list and use `b-results-list` for styling the block

```suggestion
const BLOCK_CLASS_NAME = "b-results-list";
```